pipeline/lovd/separateFunctionalAnalysisTechniqueAndResult.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+"""
+Zack Fischmann
+9/21/2018
+
+Description:
+    LOVD functional analysis technique and result are reported in a colon delimited fashion
+    e.g. technique:result. This file splits them into two separate fields.
+"""
+
+import argparse
+import sys
+import os
+import urllib
+from collections import defaultdict
+
+
+def parse_args():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input', type=argparse.FileType('r'),
+                        help='Input LOVD file for ammendment.')
+    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+                        help='Ouput txt file result.')
+    options = parser.parse_args()
+    return options
+
+
+def main(args):
+    options = parse_args()
```

Comment:
An interesting point -- looking through our python scripts there are many different ways we pass arguments, most of which are sloppy and/or verbose. I think most of them are perpetuated as a result of copy pasta (I know that's how I start all my scripts).

What would be an optimal setup in your opinion? Simply removing `args` here and updating the initial invocation as follows works fine:
```
if __name__ == "__main__":
    main()
```

Reply:
Yeah, I think it's reasonable to just not pass the arguments into main() since argparse reads from sys.argv by default, and afaik that's how most people use it.